### DISCUSSION TITLE
Where can I learn more about MONAI Bundle?

### DISCUSSION THREAD

**Discussion by ebrahimebrahim** [Project-MONAI/MONAI#5232] [Q&A] [answered]
All I've found are the [spec](https://docs.monai.io/en/latest/mb_specification.html) and [this config syntax document](https://docs.monai.io/en/latest/config_syntax.html). I also know I can find many examples at the [model zoo](https://monai.io/model-zoo.html).

But I still don't really understand what's going on in the [config syntax document](https://docs.monai.io/en/latest/config_syntax.html).

It looks like there's an entire little language built into the json config files. Like a restricted version of python where you can specify imports, instantiate objects, evaluate expressions, do some text replacements, etc. Where would one learn how to use this little language?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @ebrahimebrahim ,

Thanks for your interest here.
You can find more introduction and detailed examples: https://github.com/Project-MONAI/tutorials/tree/main/bundle

Thanks.

**Reply by ebrahimebrahim** [reply]
Thanks! This is just what I was looking for.

I think it would be helpful if [this page](https://docs.monai.io/en/latest/bundle_intro.html) (or some similar "landing page") had some more high level introductory text on it, maybe also linking to the [bundle tutorials](https://github.com/Project-MONAI/tutorials/tree/main/bundle) or to the [getting started notebook](https://github.com/Project-MONAI/tutorials/blob/main/bundle/get_started.ipynb) in there. At the moment what can be straightforwardly found by googling is the [spec](https://docs.monai.io/en/latest/mb_specification.html) and [the config syntax document](https://docs.monai.io/en/latest/config_syntax.html), both of which are great resources but poor introductions to just quickly figure out what it's all about.

**Reply by Nic-Ma** [reply]
Hi @ebrahimebrahim ,

Thanks for your feedback, I created a PR to enhance the doc page: https://github.com/Project-MONAI/MONAI/pull/5234.
